ocis/pkg/command/migrate.go

```diff
@@ -275,6 +275,13 @@ func revaPublicShareConfig(cfg *sharing.Config) map[string]interface{} {
 			"file":         cfg.PublicSharingDrivers.JSON.File,
 			"gateway_addr": cfg.Reva.Address,
 		},
+		"jsoncs3": map[string]interface{}{
+			"gateway_addr":        cfg.PublicSharingDrivers.JSONCS3.ProviderAddr,
```

Comment:
doesn't that neet the real gateway?
```suggestion
			"gateway_addr":        cfg.Reva.Address,
```